### ISSUE TITLE
Roll-up report is empty for multiple site reports

### ISSUE THREAD

**Issue by TWoofenden** [seanhamlin/site-audit#209]
**Composer Info**
```
Please provide the output from running `composer info`.
```
acquia/http-hmac-php                  4.0.1   An implementation of the HTTP HMAC Spec in PHP ...
caseyamcl/toc                         v1.2    Simple Table-of-Contents Generator for PHP.  Ge...
cocur/slugify                         v1.4.1  Converts a string into a slug.
composer/semver                       1.4.2   Semver library that offers utilities, version c...
doctrine/annotations                  v1.6.0  Docblock Annotations Parser
doctrine/lexer                        v1.0.1  Base library for a lexer that can be used in To...
drutiny/acquia                        2.4     Acquia specific policies and audits for Drutiny.
drutiny/cloudflare                    2.1     Integration with Cloudflare
drutiny/drutiny                       2.3.1   This is a generic Drupal 7 and Drupal 8 site au...
drutiny/http                          2.4     Drutiny auditing and policies on HTTP requests
drutiny/plugin-drupal-7               2.1     Drupal 7 plugin for Drutiny
drutiny/plugin-drupal-8               2.1     Drupal 8 plugin for Drutiny
drutiny/sumologic                     2.3     Integration with SumoLogic
erusev/parsedown                      1.7.1   Parser for Markdown.
fiasco/symfony-console-style-markdown 1.0     A Symfony Console Style extension to support Ma...
guzzlehttp/guzzle                     6.3.3   Guzzle is a PHP HTTP client library
guzzlehttp/promises                   v1.3.1  Guzzle promises library
guzzlehttp/psr7                       1.4.2   PSR-7 message implementation that also provides...
kevinrob/guzzle-cache-middleware      v3.2.1  A HTTP/1.1 Cache for Guzzle 6. It's a simple Mi...
knplabs/knp-menu                      2.3.0   An object oriented menu library
masterminds/html5                     2.3.0   An HTML5 parser and serializer.
mustache/mustache                     v2.12.0 A Mustache implementation in PHP.
paragonie/random_compat               v2.0.17 PHP 5.x polyfill for random_bytes() and random_...
psr/cache                             1.0.1   Common interface for caching libraries
psr/http-message                      1.0.1   Common interface for HTTP messages
psr/log                               1.0.2   Common interface for logging libraries
psr/simple-cache                      1.0.1   Common interfaces for simple caching
romaricdrigon/metayaml                1.1     Using [Yaml|Xml|json] schemas files to validate...
symfony/cache                         v3.4.15 Symfony Cache component with PSR-6, PSR-16, and...
symfony/class-loader                  v3.4.15 Symfony ClassLoader Component
symfony/console                       v3.4.15 Symfony Console Component
symfony/debug                         v4.1.4  Symfony Debug Component
symfony/expression-language           v3.4.15 Symfony ExpressionLanguage Component
symfony/finder                        v3.4.15 Symfony Finder Component
symfony/polyfill-apcu                 v1.9.0  Symfony polyfill backporting apcu_* functions t...
symfony/polyfill-ctype                v1.9.0  Symfony polyfill for ctype functions
symfony/polyfill-mbstring             v1.9.0  Symfony polyfill for the Mbstring extension
symfony/process                       v3.4.15 Symfony Process Component
symfony/translation                   v4.1.4  Symfony Translation Component
symfony/validator                     v3.4.15 Symfony Validator Component
symfony/yaml                          v3.4.15 Symfony Yaml Component
twig/twig                             v1.35.4 Twig, the flexible, fast, and secure template l...
**PHP Info**
```
Please provide the output from running `php --version`.
```
PHP 7.1.16

**To Reproduce**
The command run to produce the error:
```
./vendor/bin/drutiny profile:run d8 aht:@prologis.prod -f html --uri=prologis.com --uri=prologisgermany.de
```
**Error output**

```
Drupal 8 Site Audit
Report audits policies over 2 sites.

Summary
Site results by policy

```

**Expected behavior**
Expected result is a roll-up report containing data on the two sites.

**Screenshots**

**Additional context**
Seeing the same thing when using the --domain source parameter; also seeing this for both ACSF and standard multisite cases. Note that the individual reports are generated as expected.

Using Drutiny 2.3.1 on OSX 10.13.6, via Drutiny CS Adaptor.

**Comment by fiasco** [comment]
Confirmed this is also happening with the test profile:
```
$ drutiny profile:run test @none --uri=test.com --uri=foo.com -f html
```

**Comment by fiasco** [comment]
This should now be fixed in a08adb44952c96558f8745b8e900df0c6885b7d0
